public/images/types/pipeline.svg

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
@eXistenZNL I think 
![image](https://cloud.githubusercontent.com/assets/6495166/22285212/fe70c0a6-e2ea-11e6-9e57-91a3a82a7584.png)
is the better alternative, will implement